java/ql/lib/semmle/code/java/dataflow/internal/DataFlowUtil.qll

```diff
@@ -164,7 +164,7 @@ predicate simpleLocalFlowStep(Node node1, Node node2) {
     pragma[only_bind_out](node2.getEnclosingCallable())
 }
 
-private newtype TContent =
+newtype TContent =
```

Comment:
Let's keep this private. The necessary parts are already exposed through `FieldContent.getField()` and `SyntheticFieldContent.getField()`.

Reply:
Oh right! I will change it!